spec/unit/serialize/v2/http/user_spec.rb

```diff
@@ -52,7 +52,7 @@
         'synced_at' => json_format_time(Time.now.utc - 1.hour),
         'correct_scopes' => true,
         'created_at' => json_format_time(Time.now.utc - 2.hours),
-        'first_logged_in_at' => json_format_time(Time.now.utc - 90.minutes),
+        'first_logged_in_at' => json_format_time(Time.now.utc - 4.hours),
```

Comment:
This will surely work… I realised after your question about why 4 hours yesterday that you were probably talking about how it didn’t make sense that `first_logged_in_at` could be _before_ `created_at`. So I wanted the test values to make more narrative sense. But maybe `90.minutes` doesn’t work as expected? That’s how it seemed. I was going to try `1.5.hours` instead but if you think it’s not important to have sensible test data, we can just merge this once it passes hehe

Reply:
Ah, I matched the 4 hours with with the stubs had, and another spec.

I also eventually got why you picked 4 hours, or any difference.